dist/panel-triggers/module.js

```diff
@@ -9784,7 +9784,13 @@ var AlertCard = /** @class */ (function (_super) {
         };
         _this.onLinkIconClick = function (event, url) {
             event.stopPropagation();
-            window.top.location.href = url;
+            // Add 'foldTabRow' query param so that Grafana tab row panel keeps folded
+            var urlBase = url.substring(0, url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var urlQuery = url.substring(url.indexOf('?') > -1 ? url.indexOf('?') : url.length);
+            var queryObj = _this.parseParamsStringToObject(urlQuery);
```

Comment:
I've seen this kind of three code lines repeated alot recently. Should there be a common function for parsing URL into an object that provides nice structure for accessing URL parts? I would also imagine there already being some library for that.

Reply:
I'll find a JS library for url parsing from npm